### PR TITLE
Enable configuration of post processors using parameters

### PR DESCRIPTION
**Pull request by notrix**

### CODE REVIEW

**Comment by darceee2** [comment]
:+1:

**Comment by antoligy** [comment]
:+1: Could play interestingly with this - https://github.com/liip/LiipImagineBundle/pull/717

**Comment by shordeaux** [comment]
👍

File: composer.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
         "symfony/form": "~2.3|~3.0",
         "symfony/console": "~2.3|~3.0",
         "symfony/phpunit-bridge": "~2.3|~3.0",
+        "symfony/process": "~2.3|~3.0",
```
Comment (lsmith77) [inline composer.json]: why is this necessary?

**Comment by lsmith77** [comment]
needs a rebase .. also a small entry to the docs is needed. sorry for the delay

**Comment by antoligy** [comment]
Separately to this, I wonder if it's worth applying the same default configuration to the other two PostProcessors, and also if it's worth implementing the ConfigurablePostProcessorInterface to these?

File: composer.json (hunk 1 above)
Comment (notrix) [inline composer.json]: Post processors use ProcessBuilder class from symfony/process component. Maybe it should be included in composer require section?

**Comment by lsmith77** [comment]
sounds good to me ..

File: composer.json (hunk 1 above)
Comment (lsmith77) [inline composer.json]: yeah .. should be in the require section

**Comment by lsmith77** [comment]
there is overlap with #756 and this PR

**Comment by antoligy** [comment]
Looks like that's been taken care of in that PR, leaving only one item for the to-do list. 😄 

@notrix If you haven't the time to write docs atm, I'm happy to write up some docs and PR against your local repo, leaving just the rebase against master before this can be merged - Sound okay?

**Comment by notrix** [comment]
@antoligy it would be great if you could write some docs. Thnx

**Comment by antoligy** [comment]
Done! https://github.com/notrix/LiipImagineBundle/pull/1

**Comment by lsmith77** [comment]
great .. github tells me that there are conflicts that must be resolved. could you rebase on master?

**Comment by notrix** [comment]
This is madness. I have merged with liip:master branch and got additional commit to this PR. So I rebased and now it shows 31 files modified.. Is it possible to fix this mess?

**Comment by antoligy** [comment]
In this case it might be worth raising a second PR which resolves this one.

Ideally after rebasing you'd want to force push to your branch, but it
sounds like it might be a little late for that!

**Comment by lsmith77** [comment]
@notrix generally you want to rebase feature branches from master instead of merging from master:
http://symfony.com/doc/current/contributing/code/patches.html#rebase-your-patch

question is if #756 covers everything in this PR?

**Comment by antoligy** [comment]
#756 doesn't include the parameters, although has now introduced a merge conflict with the constructor.  I'll address that in turn - For the meantime, I've rebased and have created #759.
